gui/main.cpp

```diff
@@ -2,8 +2,8 @@
 #include <iostream>
 #include <thread>
 
+#include "./Network/Network.hpp"
```

Comment:
No relative path if possible
```suggestion
#include "Network/Network.hpp"
```

Reply:
Done